cmd/terramate/e2etests/list_watch_test.go

```diff
@@ -127,6 +128,72 @@ func TestListWatchNonExistentFile(t *testing.T) {
 	assertRun(t, cli.listChangedStacks())
 }
 
+func TestListWatchElementsWithFuncalls(t *testing.T) {
+	s := sandbox.New(t)
+
+	extDir := s.RootEntry().CreateDir("EXTERNAL")
+	extFile := extDir.CreateFile("FILE.TXT", "anything")
```

Comment:
FILE.TXT made me think about old Windows stuff :rofl:

Reply:
I think it is the old msdos stuff, everything always showed on uppercase..or something like that :laughing: